addons/calendar/controllers/main.py

```diff
@@ -66,6 +66,7 @@ def view_meeting(self, token, id, **kwargs):
         timezone = attendee.partner_id.tz
         lang = attendee.partner_id.lang or get_lang(request.env).code
         event = request.env['calendar.event'].with_context(tz=timezone, lang=lang).sudo().browse(int(id))
+        company = request.env['res.users'].sudo().search([('id', '=', event.user_id.id or event.create_uid.id)]).company_id
```

Comment:
why do a search when you can just use the existing values ?
```suggestion
        company = event.user_id and event.user_id.company_id or event.create_uid.company_id
```